src/Files.App/Utils/Storage/Operations/FileOperationsHelpers.cs

```diff
@@ -1018,5 +1018,27 @@ protected override void Dispose(bool disposing)
 				}
 			}
 		}
+
+		public static string GetIncrementalName(bool overWriteOnCopy,string? filePathToCheck)
```

Comment:
A space is needed after the coma:
```suggestion
		public static string GetIncrementalName(bool overWriteOnCopy, string? filePathToCheck)
```